python/ray/ml/train/data_parallel_trainer.py

```diff
@@ -0,0 +1,184 @@
+import logging
+from typing import Dict, Callable, Optional, Union
+
+from ray.ml.trainer import Trainer
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.train.trainer import GenDataset
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+from ray.train import BackendConfig
+from ray.util.annotations import DeveloperAPI
+
+logger = logging.getLogger(__name__)
+
+
+@DeveloperAPI
+class DataParallelTrainer(Trainer):
+    """A Trainer for data parallel training.
```

Comment:
Just wondering, are these docs being compiled anywhere? Or will that be verified in a separate PR?

Reply:
That will be in a separate PR.